px-data-grid.html

```diff
@@ -487,6 +485,15 @@
           }
         }
 
+        _handleColumnFroze(event) {
+          const columnsRow = this._vaadinGrid._columnTree[this._vaadinGrid._columnTree.length - 1].slice(0);
+          const dataColumns = columnsRow.filter(c => c.isDataColumn);
+          if (dataColumns.length) {
+            this._vaadinGrid.insertBefore(event.detail.column, dataColumns[0]);
+            this._vaadinGrid._observer.flush();
```

Comment:
We can remove that.

Reply:
Why? what? sorry do not understand this.

Reply:
the observer flush... o yeah, will do that